package.json

```diff
@@ -8,6 +8,7 @@
   "scripts": {
     "build": "tsc",
     "build:watch": "tsc --watch",
+    "clean": "rm -rf lib test my-element.{d.ts,d.ts.map,js,js.map}",
```

Comment:
I'm not sure we want to remove all of test. I often have non-.ts files (like .html and data files) checked in there.

Could we use `git clean` here instead?

Reply:
We could--although I think it'll remove `node_modules` as well, which is not ideal.

Reply:
I looked at the .gitignore file when deciding what to remove, but clearly if there are non-generated files in test, we shouldn't remove the directory.

Reply:
Also, if you add files to `test` and run `npm clean` before committing them, `git clean` would remove those, too, which might be a little surprising. Perhaps best to explicitly remove just the generated files?